opal/mca/common/ucx/common_ucx_wpool.c

```diff
@@ -32,6 +32,8 @@ __thread int initialized = 0;
 #endif
 
 bool opal_common_ucx_thread_enabled = false;
+int64_t opal_common_ucx_ep_counts = 0;
```

Comment:
what about I make the increment/decrement an atomic operation? any thoughts on how I can let increment/decrement happen only if we have a debug build to avoid the performance side-effects?

Reply:
Yes, they need to be atomic. And you can check for `OPAL_ENABLE_DEBUG` to see if debug mode is enabled.

Reply:
@devreal Thanks for the note. Please take a look at the updated commit.